src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -178,52 +158,21 @@ def cli_cosmosdb_update(client,
         consistency_policy = ConsistencyPolicy(default_consistency_level=default_consistency_level,
                                                max_staleness_prefix=max_staleness_prefix,
                                                max_interval_in_seconds=max_interval)
-    else:
-        consistency_policy = existing.consistency_policy
-
-    if not locations:
-        locations = []
-        for loc in existing.read_locations:
-            locations.append(
-                Location(location_name=loc.location_name,
-                         failover_priority=loc.failover_priority,
-                         is_zone_redundant=loc.is_zone_redundant))
-
-    if ip_range_filter is None:
-        ip_range_filter = existing.ip_range_filter
-
-    if enable_automatic_failover is None:
-        enable_automatic_failover = existing.enable_automatic_failover
 
-    if enable_virtual_network is None:
-        enable_virtual_network = existing.is_virtual_network_filter_enabled
-
-    if virtual_network_rules is None:
-        virtual_network_rules = existing.virtual_network_rules
-
-    if tags is None:
-        tags = existing.tags
-
-    if enable_multiple_write_locations is None:
-        enable_multiple_write_locations = existing.enable_multiple_write_locations
-
```

Comment:
Why do you delete this part?

Reply:
Originally, we were using "put" to update the account, and now we are using "patch". With "patch", null values are ignored in our RP so we do not need to copy over the existing value of each property. All this code was used to ensure that an account update (via "put") did not incorrectly reset a property to the default value.